hw/top_earlgrey/data/xbar_main.hjson

```diff
@@ -8,13 +8,27 @@
   reset_primary: "rst_main_ni", // Main reset, used in sockets
   other_reset_list: [ "rst_fixed_ni", "rst_spi_host0_ni", "rst_spi_host1_ni", "rst_usb_ni"] // Secondary clocks used by specific nodes
 
+  // Rationale for pipeline and req/rsp_fifo_pass:
```

Comment:
@eunchan 
@msfschaffner 
let me know if this makes sense to you.

@jdonjdon i should probably follow on this change with another change in `flash_ctrl`...